src/main/content/architect.html

```diff
@@ -20,9 +20,10 @@
         <div class="col text-center">
             <div>
                 <p id="intro-text">
-                  The solution architect (or someone serving in that role) provides governance over the set of stacks available for developers 
-                  to seamlessly use, empowering developers with agility and speed while complying with company policies. This management CLI and 
-                  access to the correct GitHub repositories enable this governance.
+                  The solution architect's role is in charge of governance over the set of stacks available for developers 
+                  to seamlessly use. This empowers developers with agility and speed while complying with company policies. The Management CLI and 
```

Comment:
Change first two sentences to:
Solution architects work with operations and development staff to define and produce a set of stacks for developers to use. This architecture allows developers to work with agility and speed while complying with company policies

Reply:
In next sentence, make capitalized words (Management and Console) lower case.